content/docs/user-guide/project-structure/configuration.md

```diff
@@ -360,6 +365,12 @@ Composition].
 
 ## state
 
+<admon type="warn">
+
+This section is obsolete. Modifying these config options will have no effect.
+
+</admon>
+
```

Comment:
This is interesting. In the past, I think we have deleted sections like this where they are no longer relevant, although I'm fine with doing it this way.

Reply:
Yeah, we did. But I'm wondering if that was correct considering that our doc engine has no support for different doc versions. Might delete it anyways. WDYT?

Reply:
If we bother to include it, let's include a version number (is it 2.50.0?). Otherwise, it feels pointless to leave the old info.

Reply:
@dberenbaum Good point. Added.